configs/sst_virtualization-virt.yaml

```diff
@@ -0,0 +1,97 @@
+document: feedback-pipeline-workload
+version: 1
+data:
+  name: Virtualization on RHEL - KVM Userspace
+  description: Userspace agents and tools that enable virtualization capabilities on RHEL
+  maintainer: sst_virtualization
+
+  modules_enable:
+  - virt:rhel
+
+  packages:
+  - amtterm
+  - augeas
+  - dtc
+  - edk2
+  - fakechroot
+  - fakeroot
+  - fuse-sshfs
+  - gperftools
+  - gtk-vnc
+  - hexedit
+  - hivex
+  - icoutils
+  - ipxe
+  - kernel
+  - libdnet
+  - libguestfs
+  - libguestfs-winsupport
+  - libiscsi
+  - libmspack
+  - libnbd
+  - libosinfo
+  - libunwind
+  - libvirt
+  - libvirt-dbus
+  - libvirt-glib
+  - libvirt-python
+  - libvirt-snmp
+  - mdevctl
+  - mingw-binutils
+  - mingw-crt
+  - mingw-filesystem
+  - mingw-gcc
+  - mingw-headers
+  - mingw-pkg-config
+  - mingw-srvany
+  - mingw-w64-tools
+  - nbdkit
+  - netcf
+  - ocaml
+  - ocaml-calendar
+  - ocaml-camlp4
+  - ocaml-cppo
+  - ocaml-csv
+  - ocaml-curses
+  - ocaml-extlib
+  - ocaml-fileutils
+  - ocaml-findlib
+  - ocaml-gettext
+  - ocaml-labltk
+  - ocaml-libvirt
+  - ocaml-ocamlbuild
+  - ocaml-srpm-macros
+  - ocaml-xml-light
+  - osinfo-db
+  - osinfo-db-tools
+  - OVMF
+  - perl-Sys-Virt
+  - phodav
+  - po4a
+  - python-augeas
+  - qemu-guest-agent
+  - qemu-kvm
+  - rhevm-guest-agent
+  - seabios
+  - sgabios
+  - SLOF
+  - spice-html5
+  - spice-xpi
+  - supermin
+  - vhostmd
+  - virt-manager
+  - virt-p2v
```

Comment:
`virt-p2v` is shipped as a separate ISO.  We don't normally list it as a separate package (eg. it doesn't appear in the RHEL 8 virt module).

Reply:
Is it a rpm though ? Or only an ISO ?
This is just to determine whether we want a package shipped in RHEL-9, and not specifically the virt module.

Reply:
It's both in fact.  We ship an ISO, but also an RPM containing the tools to make the ISO, both called `virt-p2v`.